src/utils/sendEmail.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Takes an email and sends a purchase receipt to that email, including
+ * each item's name, quantity, and price (individual price * quantity), date
+ * of purchase, and order/purchase number
+ * 
+ * Parameters:
+ *    String email: email to send receipt to
+ *    Array item: array of arrays: [name, qty, price]
```

Comment:
Might be better to use a JSON format instead of an array format. For instance, you can have an array of {name: string, qty: number, price: string}. Then you can access an item's attributes by calling items[0].name, items[0].qty, items[0].price. Would make the code a lot more readable and bug-fixable instead of accessing data by just getting an index